release-notes.md

```diff
@@ -51,6 +51,14 @@ This document contains release notes per zib, indicating differences with their
 * The extension EpisodOfCare-Title has been replaced by ext-EpisodeOfCare.EpisodeOfCareName because this zib concept is functionally equivalent.
 * The extensions EpisodeOfCare-DateFirstEncounter and EpisodeOfCare-DateLastEncounter from the previous profile are not inherited because no functional backing exists.
 
+# zib-FLACCpainScale
+* The system and code value has changed to `http://snomed.info/sct` and `108291000146105` on `Observation.code`.
+* The system and code value has changed to `http://snomed.info/sct` and `12013003` on `Observation.component:face.code`.
+* The system and code value has changed to `http://snomed.info/sct` and `12013004` on `Observation.component:legs.code`.
```

Comment:
I can't find the 12013003 code on snomed: https://terminologie.nictiz.nl/art-decor/snomed-ct?conceptId=12013004

It's deprecated. Is this a known issue at the zib centre?

Reply:
Created a ticket https://bits.nictiz.nl/browse/ZIB-1628

Reply:
The codes are from here: https://zibs.nl/wiki/Zib_Codestelsels#Codesysteem_ScoreObservaties changed the system value